lib/spans/span-context.js

```diff
@@ -1,13 +1,36 @@
 'use strict'
 
+const {MAXIMUM_CUSTOM_ATTRIBUTES} = require('../attributes')
+const {PrioritizedAttributes, ATTRIBUTE_PRIORITY} = require('../prioritized-attributes')
+const {DESTINATIONS} = require('../config/attribute-filter')
+
+// Scoping impacts memoization. We could decide to add a scope instead of including
+// spans in segment scope in the future.
+const ATTRIBUTE_SCOPE = 'segment'
```

Comment:
Do you think it's worth adding 'segment' and 'transaction' scope to DESTINATIONS?

Reply:
i considered but its sort of a different type of "destination".  i'm not sure. honestly, its sort of awkward and just a means of choosing a filter. there's prob a better approach separate from both of these.